lib/livebook/config.ex

```diff
@@ -760,6 +765,15 @@ defmodule Livebook.Config do
           abort!("module given as custom identity provider in #{env} could not be found")
         end
 
+      "livebook_teams" ->
+        case identity_provider_type_to_module() do
+          %{"livebook_teams" => module} ->
+            {:zta, module, nil}
+
+          _ ->
+            abort!("invalid configuration for identity provider given in #{env}")
+        end
+
```

Comment:
The user will never set the identity provider directly. Today we already do this in application.ex:

```
Application.put_env(:livebook, :teams_auth?, true)
```

My suggestion is to change the line above to this:

```elixir
Application.put_env(:livebook, :teams_auth, :online | :offline)
```

Now, in this code, we check for the teams_auth if no env var is given. I can push some commits to main soon.